npm-shrinkwrap.json

```diff
@@ -540,7 +540,7 @@
     },
     "@types/ws": {
       "version": "4.0.2",
-      "resolved": "https://registry.npmjs.org/@types/ws/-/ws-4.0.2.tgz",
+      "resolved": "http://registry.npmjs.org/@types/ws/-/ws-4.0.2.tgz",
```

Comment:
We have a couple of http resolutions here too. :/

Reply:
Ref: #6577